tests/pvc_utils.go

```diff
@@ -0,0 +1,74 @@
+package tests
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	defaultTimeout = 30*time.Second
+)
+
+// Creates a standard PVC in the passed in namespace.
```

Comment:
You can browse k8s pv/pvc util funcs [here](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go). Some useful ones may be:
```
WaitForPersistentVolumeClaimPhase
WaitForPersistentVolumeClaimDeleted

```
Have you thought about us vendoring in k8s' e2e framwork pkg? Is this too much?

Reply:
we probably don't want to vendor kubernetes packages directly. Anything that's meant for consumption has been broken out into its own repos with its own releases.

Reply:
Right we don't want to vendor in the k8s packages, but we will likely fairly wholesale steal entire utility function files. for instance I am fairly certain I want more or less the entire https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/exec_util.go. But I will add that when I do the Framework part. As for the PVC util functions, let me double check, I looked at a few and they seemed a little too heavy for what we wanted.

Reply:
@jeffvance So I checked out the methods in the k8s framework, we don't want to create/delete PVs as we assume they exist. The create/delete PV in this PR are basically identical, except mine handles api errors better. That leaves the POD utility methods, which I think are going to be valuable, but I would like to attack those when I add the Framework.